### DISCUSSION TITLE
Is there a way to set the page title in the browser dynamically

### DISCUSSION THREAD

**Discussion by zoechi** [DioxusLabs/dioxus#985] [General]
with and without the router

**Comment by mrxiaozhuox** [comment]
## For Dioxus Web

I'm using `js_sys` to eval a javascript code to change the title.

```
js_sys::eval("document.title = 'XXX'");
```

or you can use `web_sys`, maybe it have wrapped function.

**Reply by zoechi** [reply]
Thanks

**Comment by CristianCapsuna** [comment]
At the time of this comment, you can do it with [this](https://docs.rs/dioxus-document/latest/dioxus_document/fn.Title.html) Dioxus native function.
